src/UseCases/FeeChange/ShowFeeChangePresenter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\MembershipContext\UseCases\FeeChange;
+
+interface ShowFeeChangePresenter {
+	public function showFeeChange( int $externalMemberId, int $currentAmountInCents, int $suggestedAmountInCents, int $currentInterval ): void;
+
+	public function showFeeChangeError(): void;
+
+	public function showFeeChangeAlreadyFilled(): void;
```

Comment:
I have a question about this presenter interface which I'm currently trying to implement in the `fundraising-application`.
What are the functions supposed to do?
Currently we mostly use HTMLPresenters that return arrays, strings, or JSONPresenters that return JSON.
What was the thought in this case here?